newIDE/app/src/MainFrame/index.js

```diff
@@ -1668,6 +1674,38 @@ const MainFrame = (props: Props) => {
       message: 'Update available',
     });
 
+  // Temporary keyboard shortcut for command palette
```

Comment:
Let's remove the comment about this being "Temporary".
First because often temporary stuff tends to become permanent :D And second because "MainFrame" does not care at all if it's temporary or not - it's the job of the hook to set up the shortcut. So being temporary is a detail of the implementation :)